apps/src/code-studio/pd/regional_partner_contact/RegionalPartnerContact.jsx

```diff
@@ -7,6 +7,31 @@ export default class RegionalPartnerContact extends FormController {
   /**
    * @override
    */
+  serializeFormData() {
+    const dataWithDistrict = {};
+    dataWithDistrict['form_data'] = Object.assign({}, super.serializeFormData()['form_data'], this.getDistrictData());
```

Comment:
nit; this means that if `serializeFormData` were to return any ancillary data not in `'form_data'`, it would here be discarded. Something like

```jsx
const formData = super.serializeFormData();
Object.assign(formData['form_data'], this.getDistrictData());
return formData
```

Would be nicer

Reply:
Will do